chain/client/tests/catching_up.rs

```diff
@@ -709,7 +709,7 @@ mod tests {
 
             let _connectors1 = connectors.clone();
 
-            let block_prod_time: u64 = 3000;
+            let block_prod_time: u64 = 3500;
```

Comment:
Despite my best efforts the test was still flaky:
http://nayduck.eastus.cloudapp.azure.com:3000/#/test/14274